src/analysis/inner_product_space/adjoint.lean

```diff
@@ -368,10 +338,111 @@ lemma is_adjoint_pair_inner (A : E' →L[ℝ] F') :
 
 end real
 
+/-! ### Self-adjoint operators -/
+
+/-- A (not necessarily bounded) operator `A` on an inner product space is self-adjoint iff
+  `A` is equal to its adjoint. -/
```

Comment:
This comment confuses me since it requires `A` to be bounded.
```suggestion
/-- An operator `A` on an inner product space is self-adjoint iff `A` is equal to its adjoint.

TODO Extend this theory to unbounded operators. -/
```
I understand that adjoints of unbounded (i.e, densely-defined?) operators are important and that developing this theory is on your roadmap but I think the comment should to be consistent with the formal statement.

Reply:
You are completely right, this came from copying old doc-strings around. Actually this was the doc-string that motivated to do the refactor in the first place since it felt completely wrong to me: as the refactor shows the original comment was never really accurate.